.travis.yml

```diff
@@ -1,7 +1,15 @@
 language: node_js
 
 node_js:
-  - "0.10"
+  - 0.10
+
+sudo: false
+
+env:
+  global:
+    - secure: ghtK6PdDy13ruL+HGT4doVcUkl8pyzukY1gkHbkIAL8BrOc0eQdGBnAxKpPGZJv/8upWyul69DgXKg26cbS633Gfp0NjM/YPUh6wEgTg46Zjirw1ejX7xmqsQUGukZXOXNsll7/Syql+KEn8ao+JOAh+gjkFfNY2mxD6Oxs56Sc=
```

Comment:
that's done in https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/feature/4.0.0/.travis.yml, will add on 4.x release